tests/datasets/test_esri2020.py

```diff
@@ -45,14 +45,14 @@ def dataset(
         return Esri2020(root, transforms=transforms, download=True, checksum=True)
 
     def test_already_downloaded(self, tmp_path: Path) -> None:
```

Comment:
This test is normally much simpler for other datasets. For example, CDL's looks like:
```python
    def test_already_extracted(self, dataset: CDL) -> None:                              
        CDL(root=dataset.root, download=True)   
```
Could we do the same thing here?